instrumentation/jetty/jetty-11.0/javaagent/jetty-11.0-javaagent.gradle

```diff
@@ -0,0 +1,29 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_11
+}
+
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.eclipse.jetty"
+    module = 'jetty-server'
+    versions = "[11,)"
+  }
+}
+
+dependencies {
+  library group: 'org.eclipse.jetty', name: 'jetty-server', version: '11.0.0'
+  implementation project(':instrumentation:servlet:servlet-5.0:javaagent')
+  implementation project(':instrumentation:jetty:jetty-common:javaagent')
+
+  // Don't want to conflict with jetty from the test server.
+  testImplementation(project(':testing-common')) {
+    exclude group: 'org.eclipse.jetty', module: 'jetty-server'
+  }
+
+  testLibrary group: 'org.eclipse.jetty', name: 'jetty-servlet', version: '11.0.0'
+
+  latestDepTestLibrary group: 'org.eclipse.jetty', name: 'jetty-server', version: '11.+'
+  latestDepTestLibrary group: 'org.eclipse.jetty', name: 'jetty-servlet', version: '11.+'
```

Comment:
Are these needed? Using `library` and `testLibrary` already gives us `+` as latestDepTestLibrary version. Is there a need currently to limit this to `11.+`?

```suggestion
```

Reply:
Seems like not, removed.